src/charm.py

```diff
@@ -120,44 +120,32 @@ def _update_fiveg_n3_relation_data(self) -> None:
 
     def _network_attachment_definitions_from_config(self) -> list[NetworkAttachmentDefinition]:
         """Returns list of Multus NetworkAttachmentDefinitions to be created based on config."""
-        access_nad_config = {
-            "cniVersion": "0.3.1",
-            "ipam": {
-                "type": "static",
-                "routes": [
-                    {
-                        "dst": self._get_gnb_subnet_config(),
-                        "gw": self._get_access_network_gateway_ip_config(),
-                    },
-                ],
-                "addresses": [
-                    {
-                        "address": self._get_access_network_ip_config(),
-                    }
-                ],
-            },
-            "capabilities": {"mac": True},
-        }
+        if invalid_configs := self._get_invalid_configs():
+            self.unit.status = BlockedStatus(
+                f"The following configurations are not valid: {invalid_configs}"
+            )
+            return None  # type: ignore
+
+        access_nad_config = self._get_access_nad_config()
+
+        if self._get_access_interface_mtu_config() is not None:
+            access_nad_config.update({"mtu": self._get_access_interface_mtu_config()})  # type: ignore[arg-type]  # noqa: E501
```

Comment:
How about we move this to `_get_access_nad_config` just to keep the config consistent at all times?